tools/mksurfdata_esmf/src/mkagfirepkmonthMod.F90

```diff
@@ -20,9 +20,9 @@ module mkagfirepkmonthMod
 
   public  :: mkagfirepkmon       ! Set agricultural fire peak month
 
-  integer , parameter :: min_valid = 1
-  integer , parameter :: max_valid = 12
-  integer , parameter :: unsetmon = 13
+  integer , parameter :: min_valid = 1  ! month value for January
+  integer , parameter :: max_valid = 13  ! value for no agricultural fire
+  integer , parameter :: unsetmon = 14  ! maximum value expected in the output file if raw dataset includes values outside the 1 through 13 range
```

Comment:
@samsrabin I thought I'd try `unsetmon = 14` instead of removing, so as to reduce code changes to a minimum.

Reply:
Please let me know if you see other pieces of the code where I may try to include comments, keeping in mind that I am not familiar with the ESMF calls and how they work.